automation/run-local-utests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+TOXWORKDIR="/tmp/.tox-nmstate"
```

Comment:
This should be a patch in ~/.cache or a a securely created temporary directory.

Reply:
Done